lib/pedicel/version.rb

```diff
@@ -1,3 +1,3 @@
 module Pedicel
-  VERSION = '0.0.2'
+  VERSION = '0.0.3'.freeze
```

Comment:
I look forward to all string literals being frozen in Ruby 3 :slightly_smiling_face: 

http://blog.honeybadger.io/when-to-use-freeze-and-frozen-in-ruby/

> And according to Matz, all string literals will be frozen automatically in Ruby 3.